config/schema_oracle.sql

```diff
@@ -13,4 +13,12 @@ end_time TIMESTAMP,
 destination_type VARCHAR(99) NOT NULL,
 destination_url VARCHAR(99),
 status CLOB,
-PRIMARY KEY (MIGRATION_ID));
\ No newline at end of file
+PRIMARY KEY (MIGRATION_ID));
+
+-- create table box_auth_user
+create table box_auth_user (
```

Comment:
Oracle equivalents for the MySQL suggestion I made can be found, but none are elegant.  I'd suggest you look and decide whether it's helpful and which implementation would be best.

Working with schema changes in Django has been a real luxury.  It makes the migration programs for the developer and keeps track of which have been applied.  We should find an equivalent for Java.